docs/classic-ui/csrf.md

```diff
@@ -1,13 +1,180 @@
 ---
 myst:
   html_meta:
-    "description": ""
-    "property=og:description": ""
-    "property=og:title": ""
-    "keywords": ""
+    "description": "How to protect Plone against CSRF attacks."
+    "property=og:description": "How to protect Plone against CSRF attacks."
+    "property=og:title": "Cross-Site Request Forgery protection in Plone"
+    "keywords": "CSRF, security, token, protection"
 ---
 
 (classic-ui-csrf-label)=
 
 # Cross-Site Request Forgery (CSRF)
 
+Cross-Site Request Forgery (CSRF or XSRF) is a type of web attack that allows an attacker to send malicious requests to a web application on behalf of a legitimate user.
```

Comment:
Let's add the following to the Glossary as well. I anticipate we will need to refer to it elsewhere.


```
CSRF
Cross-Site Request Forgery
    Cross-Site Request Forgery (CSRF or XSRF) is a type of web attack that allows an attacker to send malicious requests to a web application on behalf of a legitimate user.
    The attack works by tricking the user's web browser into sending a request to the web application that the user did not intentionally make.
    This can allow an attacker to perform actions on the web application without the user's knowledge or consent.
    In Plone, CSRF protection is done almost transparently by [`plone.protect`](https://pypi.org/project/plone.protect/).
```

Reply:
I added an entry. Question now is, does it provide any value to references it in the csrf.md?

Reply:
Not really, because it is already explicitly defined in the text, which is good. I anticipate it will be useful elsewhere in the future.